components/select/nz-select-top-control.component.html

```diff
@@ -34,13 +39,18 @@
   </ng-container>
   <!--multiple or tags mode-->
   <ul *ngIf="nzSelectService.isMultipleOrTags">
-    <ng-container *ngFor="let option of nzSelectService.listOfCachedSelectedOption | slice: 0 : nzMaxTagCount;trackBy:trackValue;">
+    <ng-container *ngFor="let option of nzSelectService.listOfCachedSelectedOption | slice: 0 : nzMaxTagCount;trackBy:trackValue; let index = index">
       <li [@zoomMotion]
         [nzNoAnimation]="noAnimation?.nzNoAnimation"
         [attr.title]="option.nzLabel"
         [class.ant-select-selection__choice__disabled]="option.nzDisabled"
         class="ant-select-selection__choice">
-        <div class="ant-select-selection__choice__content">{{ option.nzLabel }}</div>
+        <ng-container *ngIf="nzCustomTemplate; else defaultMultipleTemplate">
```

Comment:
could simplify to
```html
<ng-container *nzStringTemplateOutlet="nzCustomTemplate;context:{nzSelectService.listOfCachedSelectedOption[index]};">
     <div class="ant-select-selection__choice__content">{{ option.nzLabel }}</div>
</ng-container>
```

Reply:
@vthinkxie nzStringTemplateOutlet doesn't seems to work with context, any ideas?

Reply:
should be
```html
<ng-container *nzStringTemplateOutlet="nzCustomTemplate;context:{$implicit: nzSelectService.listOfCachedSelectedOption[index]};">
  <div class="ant-select-selection__choice__content">{{ option.nzLabel }}</div>
</ng-container>
```

Reply:
I tried that and no lucky.
Errors:
Can't bind to 'nzStringTemplateOutletContext' since it isn't a known property of 'ng-container'.
1. If 'nzStringTemplateOutletContext' is an Angular directive, then add 'CommonModule' to the '@NgModule.imports' of this component.
2. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component. ("e.listOfCachedSelectedOption[0]?.nzLabel"
      [ngStyle]="selectedValueStyle">
      <ng-container [ERROR ->]*nzStringTemplateOutlet="nzCustomTemplate;context:{$implicit: nzSelectService.listOfCachedSelectedOpt"): ng:///NzSelectModule/NzSelectTopControlComponent.html@23:20
Property binding nzStringTemplateOutletContext not used by any directive on an embedded template. Make sure that the property name is spelled correctly and all directives are listed in the "@NgModule.declarations". ("nzSelectService.listOfCachedSelectedOption[0]?.nzLabel"

Reply:
plz rebase onto master before continue your work

Reply:
I see, it just updated 2 days ago 👍 thanks.